make.sh

```diff
@@ -523,13 +523,13 @@ test_py() {
     py_ensure_env_active
 
     # shellcheck disable=SC2086
-    python3 ${build_target_dir}/test/functional/test_runner.py \
+    (set -x; python3 ${build_target_dir}/test/functional/test_runner.py \
```

Comment:
Is this verbose intentional?

Reply:
yeah, would be useful to see what test is running, since it's just one line